<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Launching superble on firefox]]>
    </line>
    <line>
      <![CDATA[current window is{01dabdfb-c0ff-4da0-a5ad-b580638a5124}]]>
    </line>
    <line>
      <![CDATA[In login popup]]>
    </line>
    <line>
      <![CDATA[Shifting to new login window]]>
    </line>
    <line>
      <![CDATA[Scrolling to webelement 'viewMore']]>
    </line>
    <line>
      <![CDATA[After JavascriptExecutor at view more]]>
    </line>
    <line>
      <![CDATA[Logged in again with fb creds while answering the default question]]>
    </line>
    <line>
      <![CDATA[Scrolling to webelement 'viewMore']]>
    </line>
    <line>
      <![CDATA[After JavascriptExecutor at view more]]>
    </line>
    <line>
      <![CDATA[Test Complete, please refer to SuperbleScreen jpg in D drive for failed screenshots]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="56224" started-at="2017-10-02T13:58:57Z" finished-at="2017-10-02T13:59:53Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="56224" started-at="2017-10-02T13:58:57Z" finished-at="2017-10-02T13:59:53Z">
      <class name="com.vamsha.selenium.tests.Superble">
        <test-method status="PASS" signature="login()[pri:0, instance:com.vamsha.selenium.tests.Superble@366647c2]" name="login" is-config="true" duration-ms="8687" started-at="2017-10-02T19:28:57Z" finished-at="2017-10-02T19:29:05Z">
          <reporter-output>
            <line>
              <![CDATA[Launching superble on firefox]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="loginAndVerifyQ()[pri:0, instance:com.vamsha.selenium.tests.Superble@366647c2]" name="loginAndVerifyQ" duration-ms="46288" started-at="2017-10-02T19:29:05Z" finished-at="2017-10-02T19:29:52Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at com.vamsha.selenium.tests.Superble.loginAndVerifyQ(Superble.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[current window is{01dabdfb-c0ff-4da0-a5ad-b580638a5124}]]>
            </line>
            <line>
              <![CDATA[In login popup]]>
            </line>
            <line>
              <![CDATA[Shifting to new login window]]>
            </line>
            <line>
              <![CDATA[Scrolling to webelement 'viewMore']]>
            </line>
            <line>
              <![CDATA[After JavascriptExecutor at view more]]>
            </line>
            <line>
              <![CDATA[Logged in again with fb creds while answering the default question]]>
            </line>
            <line>
              <![CDATA[Scrolling to webelement 'viewMore']]>
            </line>
            <line>
              <![CDATA[After JavascriptExecutor at view more]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginAndVerifyQ -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.vamsha.selenium.tests.Superble@366647c2]" name="tearDown" is-config="true" duration-ms="1211" started-at="2017-10-02T19:29:52Z" finished-at="2017-10-02T19:29:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginAndVerifyQ status=FAILURE method=Superble.loginAndVerifyQ()[pri:0, instance:com.vamsha.selenium.tests.Superble@366647c2] output=current window is{01dabdfb-c0ff-4da0-a5ad-b580638a5124}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Complete, please refer to SuperbleScreen jpg in D drive for failed screenshots]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.vamsha.selenium.tests.Superble -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
